Resources/Prototypes/_StarLight/Entities/Objects/Weapons/Melee/cyberlimb.yml

```
@@ -50,4 +50,37 @@
     endSound:
       path: /Audio/Weapons/chainsaw.ogg
   - type: Reflect
-    reflectProb: 0.2
\ No newline at end of file
+    reflectProb: 0.2
+
+#Mantis Blade
+- type: entity
+  name: mantis blade
+  parent: [BaseItem, BaseSyndicateContraband]
+  id: CyberMantisBlade
+  description: A sharp blade that extended from your own arm, time to burn down a corporation.
+  components:
+  - type: Sprite
+    sprite: _Starlight/Objects/Weapons/Melee/cybermantisblade.rsi
+    state: cybermantisblade
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    attackRate: 2.2
+    autoAttack: true
+    damage:
+      types:
+        Slash: 18
+        Structural: 55
```

Make it more balanced, 40 slash in 1 second it's too much, also with 121 structural.
```suggestion
  - type: MeleeWeapon
    wideAnimationRotation: -135
    attackRate: 2.0
    autoAttack: false
    damage:
      types:
        Slash: 15
        Structural: 25
```